quickstart-template/webhook/README.md

```diff
@@ -3,38 +3,16 @@
 This is a template repository for quickly getting started with a Webhook App,
 which can be used for setting up Webhooks for listening to blockchain event notifications.
 
-## Local Webhook App
-
-To set up the template, run the following commands from this `webhook` folder:
-
-```bash
-npm install
-npm run start-webhook-app
-```
-
-This command will set up a webhook app locally.
```

Comment:
I don't think this should be removed, this is the webapp that should be running to receive the webhook payload. (not related with hosting solutions)

Reply:
reverted and added some context